libcobblersignatures/__init__.py

```diff
@@ -87,6 +87,8 @@ def importsignatures(self, import_type: ImportTypes, source: str):
             self._importsignaturesurl(source)
         elif import_type == ImportTypes.STRING:
             self.signaturesjson = source
+        elif import_type == ImportTypes.BUILD_IN:
```

Comment:
Typo in the name of the constant. I suppose we want `BUILT_IN`. Please also change this in other locations.